README.md

```diff
@@ -145,3 +145,38 @@ This directory will contain the request and limit information for 1Gi/2Mb.
 ```
 
 > NOTE: To aid the application, when hugepage fields are being requested via the Downward API, Network Resource Injector also mutates the pod spec to add the environment variable `CONTAINER_NAME` with the container's name applied.
+
+### Node Selector
+If a ```NetworkAttachmentDefinition``` CR annotation ```k8s.v1.cni.cncf.io/nodeSelector``` is present and a pod utilizes this network, Network Resources Injector will add this node selection constraint into the pod spec field ```nodeSelector```. Injecting a single node selector label is currently supported.
+
+Example:
+```yaml
+apiVersion: k8s.cni.cncf.io/v1
+kind: NetworkAttachmentDefinition
+metadata:
+  name: test-network
+  annotations:
+    k8s.v1.cni.cncf.io/resourceName: example.com/foo
```

Comment:
@martinkennelly Overall it looks good.  just a minor comment here. can we remove resourceName from the example ? I think this may not be needed for macvlan network type attachments.

Reply:
ah yes - corrected!